test/e2e/kubectl/kubectl.go

```diff
@@ -857,12 +860,16 @@ metadata:
 					{"Annotations:"},
 					{"Status:", "Running"},
 					{"IP:"},
-					{"Created By:", "ReplicationController/redis-master"},
+				}
+				if !gte {
+					requiredStrings = append(requiredStrings, []string{"Created By:", "ReplicationController/redis-master"})
+				}
+				requiredStrings = append(requiredStrings, [][]string{
 					{"Controlled By:", "ReplicationController/redis-master"},
 					{"Image:", redisImage},
 					{"State:", "Running"},
 					{"QoS Class:", "BestEffort"},
-				}
+				}...)
```

Comment:
From `checkOutput` code, I don't think the order matters. Would you check again? If so, we can just append it at the end.

Reply:
The order matters here.
The index of the actual output always increases after a match, so the order matters.
https://github.com/mengqiy/kubernetes/blob/3d5ff42cc5cc0a71668e9490ae8cb14b644d02e7/test/e2e/kubectl/kubectl.go#L1759

Reply:
ok